source/buildings/university.md

```diff
@@ -53,6 +53,6 @@ Page 2 of the GUI will show you the options for each research tree.
     
 <br>
     
-Once you click on a research tree, you will see the options for each research branch. Each option will tell you the requirements and how long it takes to research that option. (Time is real-world time spent in-game.)
+Once you click on a research tree, you will see the options for each research branch. Each option will tell you the requirements and how long it takes to research that option. (Time is real-world time spent in-game. However, Researchers will sometimes use offline time to work on researches. Researchers' Mana skill affects the max amount of offline time they can use, and their Knowledge skill affects the amount of research hours they get from one hour of offline time.)
```

Comment:
I mean the thing's measured in seconds, so hours have very little to do with it. Generally you don't put a complete concept in brackets. Check out my original suggestion, you used a similar phrasing for the researcher now anyway.